evap/staff/tests/test_importers.py

```diff
@@ -372,7 +402,37 @@ def test_replace_consecutive_and_trailing_spaces(self):
         )
 
         success_messages, __, __ = EnrollmentImporter.process(excel_content, self.semester, None, None, test_run=True)
-        self.assertIn("The import run will create 1 courses/evaluations and 3 users", "".join(success_messages))
+        self.assertIn("The import run will create 1 course/evaluation and 3 users:", "".join(success_messages))
+
+    def test_enrollment_success_messages(self):
+        excel_content = excel_data.create_memory_excel_file(excel_data.test_enrollment_data_filedata)
+        success_messages, __, __ = EnrollmentImporter.process(excel_content, self.semester, None, None, test_run=True)
+        self.assertIn("The import run will create 23 courses/evaluations and 23 users:", success_messages[4])
+        success_messages, __, __ = EnrollmentImporter.process(
+            excel_content, self.semester, self.vote_start_datetime, self.vote_end_date, test_run=False
```

Comment:
It might make sense to create a `importer_args` variable and do the calls like `.process(*importer_args, test_run=True)`.